pkg/config/apiserver.go

```diff
@@ -197,15 +202,30 @@ func (fg *FeatureGates) validateFeatureGates() error {
 	if fg.FeatureSet == FeatureSetCustomNoUpgrade && len(fg.CustomNoUpgrade.Enabled) == 0 && len(fg.CustomNoUpgrade.Disabled) == 0 {
 		return fmt.Errorf("CustomNoUpgrade enabled or disabled lists must be set when FeatureSet is CustomNoUpgrade")
 	}
-	// Must not have any feature gates that are enabled and disabled at the same time
-	var illegalFeatures []string
-	for _, enabledFeature := range fg.CustomNoUpgrade.Enabled {
-		if slices.Contains(fg.CustomNoUpgrade.Disabled, enabledFeature) {
-			illegalFeatures = append(illegalFeatures, enabledFeature)
+
+	var errs = make(sets.Set[error], 0)
+	for _, requiredFG := range RequiredFeatureGates {
+		// Edge case: Users must not be allowed to explicitly disable required feature gates.
+		if sets.NewString(fg.CustomNoUpgrade.Disabled...).Has(requiredFG) {
```

Comment:
Maybe instead of prevent disabling them, we could not interfere if they're specified by user? I.e. if user wants them disabled, we don't enable them; if the don't mention them or enable them, we enable them.

Lack of UserNS FG doesn't break the cluster, just the origin tests would not pass, but it's up to the users to disable or keep it. WDYT?